.ci/openshift-ci/Dockerfile

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-# Copyright (c) 2019-2021 Red Hat, Inc.
+# Copyright (c) 2023 Red Hat, Inc.
```

Comment:
This file wasn't change, do not change the license header.

Reply:
It seems, happened a bit confusing thing..
I recently did  PR ([Create 'Dockerfile' to configure ci-operator](https://github.com/eclipse-che/che-server/pull/495)), where added this Dockerfile, but didn't take into account Valeriy's comment about header.
So I decided to fix it here.

Reply:
Ok, so it should be `# Copyright (c) 2023 Red Hat, Inc.`

Reply:
Correct header: `Copyright (c) 2019-2023 Red Hat, Inc.`

Reply:
@dmytro-ndp  If the file is new, just is created, then we'll point the only current year